common/web/types/src/keyboard-object.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ */
+import { OutputTarget } from './outputTarget.interface.js';
+import { TouchLayoutPlatform as LayoutFormFactorSpec } from './keyman-touch-layout/keyman-touch-layout-file.js';
+
+export type ComplexKeyboardStore = (string | { t: 'd', d: number } | { ['t']: 'b' })[];
+
+type KeyEvent = {};
```

Comment:
This is just a stub with a full implementation inside KeymanWeb, correct? We should document that accordingly. I think it would be good to do the same with OutputTarget, because we are leaking too many kmw abstractions into common/web/types otherwise.